frontend/src/services/request-verification-link/email-link-service.ts

```diff
@@ -0,0 +1,21 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+
+interface IEmail {
+  email: string;
+}
+
+const emailApi = createApi({
+  reducerPath: 'emailApi',
+  baseQuery: fetchBaseQuery({ baseUrl: import.meta.env.VITE_API_ORIGIN_URL }),
+  endpoints: (build) => ({
+    requestLink: build.mutation<void, IEmail>({
+      query: (email) => ({
+        url: '/user/verification',
+        method: 'POST',
+        body: email,
+      }),
+    }),
+  }),
+});
+
+export { emailApi };
```

Comment:
Please follow the one-root-createApi that @o-rudych shared here - https://binary-academy-2022.slack.com/archives/C03SA9P93C6/p1660658147966669